test/src/launcher.spec.ts

```diff
@@ -595,6 +595,30 @@ describe('Launcher specs', function () {
           await close();
         }
       });
+      it('should restore to original window size after taking fullPage screenshots when defaultViewport is null', async () => {
+        const {server, context, close} = await launch({
```

Comment:
let's add this test to test/src/screenshot.spec.ts and use getTestState instead of launching a new browser

Reply:
I guess you need to have the launch() to specify defaultViewport to be null, so disregard the part about getTestState

Reply:
yes I need to specify the defaultViewport